mitmproxy/tools/web/app.py

```diff
@@ -431,6 +435,15 @@ def get(self, flow_id, message, content_view):
         ))
 
 
+class Commands(RequestHandler):
+    def post(self):
+        result = self.master.commands.execute(self.command)
```

Comment:
Can we just access `request.json` here instead of adding the `.command` property? That's more direct and in line with the other code. We only need that specific getter here.